src/unix/os390-syscalls.c

```diff
@@ -332,18 +332,18 @@ int epoll_wait(uv__os390_epoll* lst, struct epoll_event* events,
     ev.fd = pfd->fd;
     ev.events = pfd->revents;
     ev.is_msg = 0;
```

Comment:
For simplicity, perhaps change this line to:
```suggestion
    ev.is_msg = i == list->size - 1;
```
and remove all of the other cleverness with counting?

Reply:
We wanted to check the message queue separately at the end because it's possible to get a message queue event without getting any fd events. The previous implementation would require looping through the entire `lst->items` array, and then finally handling the message queue event at the very end. With this change, we can instead skip the for loop entirely when `_NFDS(pollret) = 0` and just handle the message queue event right away.

Reply:
Okay. It seems the list would be short enough (relative to the time for the syscall especially), but with the other comment fixed, I see no reason to not use this optimization.